amazonorders/forms.py

```diff
@@ -257,12 +316,25 @@ def __init__(self,
 
     def fill_form(self,
                   additional_attrs: Optional[Dict[str, Any]] = None) -> None:
+        if not self.form:
+            raise AmazonOrdersError(
+                "Call AuthForm.select_form() first."
+            )  # pragma: no cover
```

Comment:
Remove the `# pragma: no cover` here, I'd like us to still keep track of where we're missing coverage.